app/presenters/card_presenter.rb

```diff
@@ -242,7 +240,8 @@ def include_duplicate_action?
       action_name == 'show' &&
         RadCommon::AppInfo.new.duplicates_enabled?(klass.name) &&
         instance.duplicate.present? &&
-        instance.duplicate.score.present?
+        instance.duplicate.score.present? &&
+        instance_policy.resolve_duplicates?
```

Comment:
this line is the only material change in this file, the rest is just refactoring to share code